### PR TITLE
Change overflow visible to scroll

### LINKED ISSUE #7259: When there's not enough data and HOT's parent doesn't have width, height, and overflow set up HOT's rows become misaligned on a scroll
### Description

When there's not enough data and HOT's parent doesn't have `width`, `height`, and `overflow` set up HOT's rows become misaligned on a scroll.

Remove the comment for `parent` class to see the correct behavior https://jsfiddle.net/AMBudnik/n0ef5k4g/

### Steps to reproduce

1. Scroll right

**Result**
bug

### Demo

https://jsfiddle.net/AMBudnik/fek73j4y/ 8.0.0 (broken)
https://jsfiddle.net/AMBudnik/rec8v6t3/ 7.4.2 (broken)

### Your environment
* Handsontable version: 8.0.0
* Browser Name and version: Chrome 85
* Operating System: macOS Catalina 10.15.3

### PR DESCRIPTION
**Pull request by Gwe-n**
Fix #7259

### Context
When the window has a fixed size, there's not enough data and HOT's parent doesn't have width, height, and overflow
The `overflow: visible` adds a scroll to the table

### Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. #7259 

### Checklist:

- [X] My code follows the code style of this project,
- [ ] My change requires a change to the documentation.

### CODE REVIEW

**Comment by AMBudnik** [comment]
Thank you for reporting @Gwe-n and referencing to my issue.

As each of the changes in the code should be covered by at least one test could I ask you to add one? That would help us a lot.

**Comment by wojciechczerniak** [comment]
https://codesandbox.io/s/handsontableexamples-pull-request-forked-wqp9l

Headers do not stick to the screen edge and scrollbars are shown

Wrong:
Screenshot 2020-09-17 at 15 50 04

Correct, headers are sticky to the edge, no unnecessary scrollbars.
Screenshot 2020-09-17 at 15 52 25

**Comment by Gwe-n** [comment]
@wojciechczerniak I looked at your link, and I didn't get it. Is my commit incorrect and/or causes a regression?

**Comment by AMBudnik** [comment]
Hey @Gwe-n it seems that we get this effect when we use your changes in the code.

**Comment by Gwe-n** [comment]
Are you sure the problem was not already there? 
When I inspect the deployed code, I can't find my modification, I have the impression that the deployment doesn't work.

Capture d’écran 2020-09-29 à 15 38 35

Moreover, in the ticket https://github.com/handsontable/handsontable/issues/7259 
the correct behaviour described here https://jsfiddle.net/AMBudnik/n0ef5k4g/
has a non-sticky header.

**Comment by AMBudnik** [comment]
Hi @Gwe-n please try to scroll to right using this demo https://codesandbox.io/s/handsontableexamples-pull-request-forked-hy8p8 

Here's what I get using Chrome

issue


that's why we always ask for tests and we also try to add as many scenarios as necessary.

**Comment by Gwe-n** [comment]
Hi @AMBudnik indeed, you were right. I pushed a new suggestion

Enregistrement de l’écran 2020-11-02 à 17 17 27

Please let me know if this is acceptable to you

**Comment by AMBudnik** [comment]
Thank you for sharing the update @Gwe-n

File: src/3rdparty/walkontable/test/unit/overlay/bottomLeftCorner.spec.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import Walkontable from 'walkontable/core';
+import Overlay from 'walkontable/overlay/_base';
+import BottomLeftCornerOverlay from 'walkontable/overlay/bottomLeftCorner';
+import TopLeftCornerOverlay from 'walkontable/overlay/topLeftCorner';
```
Comment (wszymanski) [inline src/3rdparty/walkontable/test/unit/overlay/bottomLeftCorner.spec.js:4]:
```suggestion
import { registerOverlayOnce, BottomLeftCornerOverlay, TopLeftCornerOverlay } from 'walkontable/overlay';
```

File: src/3rdparty/walkontable/test/unit/overlay/bottomLeftCorner.spec.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    holder.appendChild(wttable);
+    div.appendChild(holder);
+    const walkontable = new Walkontable({
+      data: {},
+      table: wttable,
+      totalRows: {},
+      totalColumns: {}
+    });
+    Overlay.registerOverlay('bottom_left_corner', BottomLeftCornerOverlay);
+    Overlay.registerOverlay('top_left_corner', TopLeftCornerOverlay);
```
Comment (wszymanski) [inline src/3rdparty/walkontable/test/unit/overlay/bottomLeftCorner.spec.js:21]: Is registration really needed, especially for `TopLeftCornerOverlay`? Btw. a way how overlays are registered has been changed. Could you adapt it for new codebase?

File: src/3rdparty/walkontable/src/overlay/bottomLeftCorner.js [hunk 3]
```diff
@@ -24,6 +25,7 @@ export class BottomLeftCornerOverlay extends Overlay {
   constructor(wotInstance) {
     super(wotInstance);
     this.clone = this.makeClone(CLONE_BOTTOM_LEFT_CORNER);
+    this.overlayRoot = null;
```
Comment (wszymanski) [inline src/3rdparty/walkontable/src/overlay/bottomLeftCorner.js:28]: Is really this change needed? I would not change it for consistency with other overlays.

**Review by wszymanski — CHANGES_REQUESTED** [review]
Hello @Gwe-n, thanks for your effort, again. It works 👏  It also have needed test 🙂  Some e2e would be also welcome, but I won't push you to doing it. However, I have some questions.

I have one needed change. Could you rename file from `bottomLeftCorner.spec.js` to `bottomLeftCorner.unit.js`. Without it, the test won't be executed.

**Comment by AMBudnik** [comment]
As there are no further updates on this pull request for months and our issues haven't been addressed, I need to close this pull request.

@Gwe-n, if you would like to continue the work, please let me know.

**Pull request closed without merging**